web/src/app/pages/main-page-container/main-page/secondary-side-panel/submission-panel/submission-panel.component.ts

```diff
@@ -35,7 +35,7 @@ export class SubmissionPanelComponent implements OnInit, OnDestroy {
 
   @Input() submissionId!: string;
   submission: Submission | null = null;
-  tasks: List<Task> | undefined;
+  tasks?: List<Task>;
```

Comment:
This shouldn't need to be nullable now?

Reply:
We can come back to this - https://github.com/google/ground-platform/issues/1466